src/i18n/strings/en_EN.json

```diff
@@ -2095,6 +2095,7 @@
     "Space used:": "Space used:",
     "Indexed messages:": "Indexed messages:",
     "Number of rooms:": "Number of rooms:",
+    "of ": "of ",
```

Comment:
suspect this should be "%s of %s" or something, or perhaps just "of". The trailing whitespace will confuse the hell out of weblate users.